src/app/services/data/version-control.service.ts

```diff
@@ -87,7 +87,8 @@ export class VersionControlService implements OnDestroy {
     this.versionsBackendService
       .createSnapshotVersion(this.variant.latestVersion.id, {
         name,
-        comment: newName ? `Neuer Name: ${newName}` : "",
+        comment: newName ? `Neuer Name: ${newName}` : "", // trad ?
```

Comment:
I wasn't sure of the purpose of it, the only mandatory field here is `id`, the rest isn't important much for the database coherence. However, it can be disturbing for a user to have one snapshot with an english comment and the next snapshot of the same variant with a german comment if the user changed the language meanwhile. I suggest to deal with this issue in https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/issues/197